<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDialog</class>
 <widget class="QDialog" name="UserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>989</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>604</width>
    <height>989</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>604</width>
    <height>989</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QQ</string>
  </property>
  <widget class="QListWidget" name="friendlistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>611</width>
     <height>871</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="headPhotolabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>60</horstretch>
     <verstretch>60</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QToolButton" name="settingButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="personalizedSignatureEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background: transparent;
    border: none;
}
QLineEdit:focus {
    border: 1px solid black; /* 设置点击时的黑框 */
}</string>
   </property>
   <property name="text">
    <string>个签</string>
   </property>
  </widget>
  <widget class="QLabel" name="nikenameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
